packages/vuepress-theme-odyssey/styles/components/_DocsPagination.scss

```diff
@@ -1,7 +1,14 @@
-.page-nav {
+.page-footer {
+  margin-top: $spacing-xl;
   padding: $spacing-m 0;
   border-top: 2px solid var(--border-color-display, #{$border-color-display});
+  text-align: center;
 
+  &,
```

Comment:
what's this for? can we nest footer inside footer?

Reply:
There are two variants of this, the pagination and standalone. In the future we should componentize footer so that this goes away. I will log a ticket and fix as a ff.